### DISCUSSION TITLE
Set the resolution of all screens when selecting an app layout

### DISCUSSION THREAD

**Discussion by BladderFox** [rxhanson/RectanglePro-Community#322] [Ideas]
Hi, recent RectanglePro customer here.

I have two desks as well as frequently working on just my MacBook while mobile. I also have different apps that I use depending on which of my jobs/hobbies I am working on.

Desktop 1 has two 1600p monitors while Desktop 2 has two 4k monitors. When I'm using my MacBook mobile I tend to use it on either 1600p or 1200p but when I'm on either desk I prefer to set the built in display to 1650x1050 in order to make the text a little bigger.

Desktop 1 monitors need to be set to 2048x1152 while I working my main job or to 1600p while studying.

Desktop 2 monitors need to be set to 1440p while I'm working, 1080p when gaming, or 4k while studying/watching video.

I have separate app layouts for each of these situations. Would it be possible to record the screen resolutions in the app layout and restore them when the app layout is selected, either by monitor connection or by the hotkey?

As a separate but associated question is it possible to only auto switch to an app layout when I connect to the displays for which the layout was created? Having two desks means that I can't currently use the auto connect switch on display connection (unless I'm just not seeing how to set it up correctly).

**Comment by rxhanson** [comment]
Thanks for the feedback. 

Adjusting display resolutions is a little out of scope on a window management app, but I can see where the lines blur here. For features like this, I typically only consider adding them in with significant demand. 

> As a separate but associated question is it possible to only auto switch to an app layout when I connect to the displays for which the layout was created?

This is more in scope, and I can see this making sense to take into account. I can add this to the backlog to look into this further.
